app/presenters/menu/default_menu.rb

```diff
@@ -141,6 +141,13 @@ def middleware_menu_section
           Menu::Item.new('middleware_server', deferred_ui_lookup(:tables => 'middleware_server'), 'middleware_server', {:feature => 'middleware_server_show_list'}, '/middleware_server'),
           Menu::Item.new('middleware_deployment', deferred_ui_lookup(:tables => 'middleware_deployment'), 'middleware_deployment', {:feature => 'middleware_deployment_show_list'}, '/middleware_deployment'),
           Menu::Item.new('middleware_datasource', deferred_ui_lookup(:tables => 'middleware_datasource'), 'middleware_datasource', {:feature => 'middleware_datasource_show_list'}, '/middleware_datasource'),
+          Menu::Item.new(
+            'middleware_messaging',
+            deferred_ui_lookup(:tables => 'middleware_messaging'),
+            'middleware_messaging',
+            {:feature => 'middleware_messaging_show_list'},
+            '/middleware_messaging'
+          ),
```

Comment:
While I think the long lines in that file are / could be an issue, please adjust to the style of the file for consistency